libs/test-env/jest.config.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable */
+export default {
+  displayName: 'test-env',
```

Comment:
how about `basename(__filename)`? but if we don't need to make more projects then maybe don't need